openid-4-verifiable-presentations-1_0.md

```diff
@@ -1611,17 +1611,30 @@ The signed request allows the Wallet to authenticate the Verifier using a trust
 
 ## Response
 
-Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. The Authorization Response is a JavaScript object, where the response parameters as defined for the Response Type are encoded as top-level members in this JavaScript object. 
+Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. The response is a JavaScript object as defined in [@!w3c.digital_credentials_api], where the Authorization Response parameters as defined for the Response Type are encoded as members within the `data` member of this JavaScript object.
```

Comment:
```suggestion
Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. This response is a JavaScript object as defined in [@!w3c.digital_credentials_api], where the OpenID4VP Authorization Response parameters as defined for the Response Type are encoded as members within the `data` member of this JavaScript object.
```

I would like to be very clear on browser API response vs openid4vp authorization response